extension/src/Xliff/XLIFFCache.ts

```diff
@@ -69,7 +62,7 @@ class XliffCache {
   }
 
   isCached(filePath: string): boolean {
-    return this.cache.has(path.basename(filePath));
+    return this.cache.get(path.basename(filePath)) !== undefined;
```

Comment:
I think you should revert to `has()` here. If we have a HUGE xliff object here, I believe that performance of `has()` is much better than `get()` - don't you think?

Reply:
Maybe. Someday I'll look into it but not today tho. Reverting :)